src/transformers/models/trocr/modeling_trocr.py

```diff
@@ -891,13 +891,40 @@ def forward(
         Example:
 
         ```python
+        >>> from transformers import TrOCRProcessor, VisionEncoderDecoderModel
```

Comment:
Would be great to avoid import `VisionEncoderDecoderModel` twice

Reply:
Good catch! Removed it